telegram/bot.py

```diff
@@ -3294,6 +3294,14 @@ def send_invoice(
     ) -> Message:
         """Use this method to send invoices.
 
+        Warning:
+            As of API 5.2 :attr:`start_parameter` is an optional argument and therefore the order
+            of the arguments had to be changed. Use keyword arguments to make sure that the
+            arguments are passed correctly.
```

Comment:
yay, now we have one warning, but what about the two shortcuts? I would warn there too